### ISSUE TITLE
composer install failure

### ISSUE THREAD

**Issue by aydot** [gourmet/muffin#1]
I got the following error while trying to add this package

```
Your requirements could not be resolved to an installable set of packages.

Problem 1
    - Installation request for gourmet/muffin ~0.1 -> satisfiable by gourmet/muffin[v0.1.0].
    - gourmet/muffin v0.1.0 requires league/factory-muffin ~2.1 -> satisfiable by league/factory-muffin[2.1.x-dev, v2.1.0, v2.1.0-RC1, v2.1.1].
    - league/factory-muffin 2.1.x-dev requires fzaninotto/faker 1.4.* -> satisfiable by fzaninotto/faker[v1.4.0].
    - league/factory-muffin v2.1.0 requires fzaninotto/faker 1.4.* -> satisfiable by fzaninotto/faker[v1.4.0].
    - league/factory-muffin v2.1.0-RC1 requires fzaninotto/faker 1.4.* -> satisfiable by fzaninotto/faker[v1.4.0].
    - league/factory-muffin v2.1.1 requires fzaninotto/faker 1.4.* -> satisfiable by fzaninotto/faker[v1.4.0].
    - Conclusion: don't install fzaninotto/faker v1.4.0
```

**Comment by aydot** [comment]
btw - i have your faker plugin installed as well, when i remove it, the composer install for the muffin package works fine.

**Comment by jadb** [comment]
Thanks for the extra info, it will help. I have just tagged all plugins in a spree and might have missed something along the way. Will investigate.

**Comment by jadb** [comment]
Can you please try again. Make sure it's a fresh install or cleanup the vendor directory and `composer.lock`.

Turns out that [league/factory-muffin:2.1/composer.json](https://github.com/thephpleague/factory-muffin/blob/2.1/composer.json#L23) explicitly requires the `1.4` version of Faker while the [gourmet/faker:1.0.2/composer.json](https://github.com/gourmet/faker/blob/1ef60bc5608f8ce40c803201662da0a3e14e6d00/composer.json#L23) was requiring `dev-master`.

**Comment by aydot** [comment]
all works now! thank you!!! and thanks for the good work too.

**Comment by aydot** [comment]
it broke my unit test though, as i was using Faker's CakePHP's ORM from the dev-master

**Comment by jadb** [comment]
Sorry. Dependencies should probably be resolved when FactoryMuffin 3.0 is released, but again, with some changes to Faker plugin too I think.
